src/main/java/appeng/parts/p2p/PartP2PFEPower.java

```diff
@@ -73,180 +65,156 @@ public void onTunnelNetworkChange()
 		this.getHost().notifyNeighbors();
 	}
 
-	@Override
-	public void onNeighborChanged( IBlockAccess w, BlockPos pos, BlockPos neighbor )
+	private IEnergyStorage getOutput()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this method into "InputEnergyStorage". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3398)